admin/sql/updates/20200630-mbs-10921-unlist-unrelated-edits.sql

```diff
@@ -0,0 +1,286 @@
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.recording
+    FROM edit_recording er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON er.edit = ed.edit
+    WHERE er.edit >= 70658947 AND er.edit < 71006471
+    AND e.type = 90
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'entity0', 'id')::int != er.recording
+    AND jsonb_extract_path_text(ed.data, 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    LIMIT 8000 -- of 805628
+)
+RETURNING *;
+
+\echo MBS-10921 (6/12): Unlist “Edit relationship” edits from recordings’ history...
+DELETE FROM edit_recording
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.recording
+    FROM edit_recording er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON er.edit = ed.edit
+    WHERE er.edit >= 70658947 AND er.edit < 71006471
+    AND e.type = 91
+    AND jsonb_extract_path_text(ed.data, 'type0') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'type1') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int != er.recording
+    AND jsonb_extract_path_text(ed.data, 'link', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    LIMIT 2600 -- of 261439
+)
+RETURNING *;
+
+\echo MBS-10921 (7/12): Unlist “Remove relationship” edits from recordings’ history...
+DELETE FROM edit_recording
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.recording
+    FROM edit_recording er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON er.edit = ed.edit
+    WHERE er.edit >= 70658947 AND er.edit < 71006471
+    AND e.type = 92
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'link', 'type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int != er.recording
+    AND jsonb_extract_path_text(ed.data, 'relationship', 'entity0', 'id')::int IN (
+        SELECT id FROM recording
+    )
+    LIMIT 150 -- of 11681
+)
+RETURNING *;
+
+\echo MBS-10921 (8/12): Unlist “Reorder relationships” edits from recordings’ history...
+DELETE FROM edit_recording
+WHERE (edit, recording) IN (
+    SELECT er.edit, er.recording
+    FROM edit_recording er
+    JOIN edit e ON er.edit = e.id
+    JOIN edit_data ed ON er.edit = ed.edit
+    WHERE er.edit >= 70658947 AND er.edit < 71006471
+    AND e.type = 99
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity0_type') = 'recording'
+    AND jsonb_extract_path_text(ed.data, 'link_type', 'entity1_type') = 'work'
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int != er.recording
+    AND jsonb_extract_path_text(data, 'relationship_order', '0', 'relationship', 'entity1', 'id')::int IN (
+        SELECT id FROM recording
+    )
```

Comment:
Same comment as for reorder relationships in edit_artist above.

Reply:
Unneeded, same as https://github.com/metabrainz/musicbrainz-server/pull/1581#discussion_r448609981.

(Fixed `s/entity1/entity0/` in https://github.com/metabrainz/musicbrainz-server/pull/1581/commits/ea6b10dd7562bed837af8b73ae7a49705bc12337 as reported in https://github.com/metabrainz/musicbrainz-server/pull/1581#discussion_r448567539.)